Build/Build.csproj

```diff
@@ -143,6 +145,7 @@
   <ItemGroup>
     <Content Include="GlobalFacetManifest.xml" />
     <Content Include="GlobalFacetManifest.NetFramework.xml" />
+    <Content Include="ProjectTypes\Analyzer.targets" />
```

Comment:
This feels incorrect? The other files targets/props files are not added as content via explicit statement.

Reply:
Yeah, rider adds it like this automatically and I overlooked it.